src/formats.rs

```diff
@@ -535,6 +535,13 @@ formats! {
     extension = "dex"
     kind = Executable
 
+    format = DashPlaylist
```

Comment:
The variant name must match the name of the file format, e.g. MpegDashMpd if the name is "MPEG-DASH MPD".